sql/core/src/test/resources/sql-tests/results/ansi/timestamp.sql.out

```diff
@@ -590,6 +590,58 @@ org.apache.spark.sql.AnalysisException
 cannot resolve '1 + (- INTERVAL '02' SECOND)' due to data type mismatch: argument 1 requires (timestamp or timestamp without time zone) type, however, '1' is of int type.; line 1 pos 7
 
 
+-- !query
+select '2011-11-11 11:11:11' - date'2011-11-11'
+-- !query schema
+struct<(2011-11-11 11:11:11 - DATE '2011-11-11'):interval day>
+-- !query output
+0 00:00:00.000000000
+
+
+-- !query
+select '2011-11-11 11:11:11' - timestamp'2011-11-11 11:11:10'
+-- !query schema
+struct<>
+-- !query output
+org.apache.spark.sql.AnalysisException
+cannot resolve '('2011-11-11 11:11:11' - TIMESTAMP '2011-11-11 11:11:10')' due to data type mismatch: argument 1 requires (timestamp or timestamp without time zone) type, however, ''2011-11-11 11:11:11'' is of string type.; line 1 pos 7
```

Comment:
why does this fail under ansi mode?

Reply:
We don't have a rule for this one

Reply:
shall we fix it under ansi mode? this is inconsistent: `string_literal - date` works but `string_literal - timestamp` does not.